Plugins/NewRelic/Source/NewRelic/ThirdParty/IOS/NewRelic.xcframework/Info.plist

```diff
@@ -8,32 +8,32 @@
 			<key>DebugSymbolsPath</key>
 			<string>dSYMs</string>
 			<key>LibraryIdentifier</key>
-			<string>ios-arm64_x86_64-maccatalyst</string>
```

Comment:
Did this change come with the new framework or did you manually edit this?

Reply:
it came with the new framework

Reply:
excellent we should be golden then